src/app/core/services/logging.service.ts

```diff
@@ -122,6 +122,9 @@ export class LoggingService {
   }
 
   debug(...params: any[]) {
+    if (AppConfig.production) {
+      return;
+    }
```

Comment:
I am thinking of removing updateLog for debug, because it is rather useless to cache debug messages in the log file. Debug messages are useful for developers who should be using the console to check for any debug messages.

Reply:
Yeahh I agree. Since debug is not useful once the app has been compiled and minified.